README.md

```diff
@@ -0,0 +1,76 @@
+# JsonApiDotNetCore MongoDB Repository
+
+Plug-n-play implementation of `IResourceRepository<TResource, TId>` allowing you to use MongoDb with your `JsonApiDotNetCore` APIs.
+
+## Installation and Usage
+
+### Models
+
+```cs
+public sealed class Book : IIdentifiable<string>
+{
+    [BsonId]
+    [BsonRepresentation(BsonType.ObjectId)]
+    [Attr]
+    public string Id { get; set; }
+
+    [Attr]
+    public string Name { get; set; }
+
+    [BsonIgnore]
+    public string StringId { get => Id; set => Id = value; }
+}
+```
+
+### Controllers
+
+```cs
+public sealed class BooksController : JsonApiController<Book, string>
+{
+    public BooksController(IJsonApiOptions options, ILoggerFactory loggerFactory, IResourceService<Book, string> resourceService)
+        : base(options, loggerFactory, resourceService)
+    {
+    }
+}
+```
+
+### Middleware
+
+```cs
+public class Startup
+{
+    public IServiceProvider ConfigureServices(IServiceCollection services)
+    {
+        services.AddSingleton<IMongoDatabase>(sp =>
+        {
+            var client = new MongoClient(Configuration.GetSection("DatabaseSettings:ConnectionString").Value);
+            return client.GetDatabase(Configuration.GetSection("DatabaseSettings:Database").Value);
+        });
+
+        services.AddScoped<IResourceRepository<Book, string>, MongoEntityRepository<Book, string>>();
```

Comment:
Also, this:
```c#
services.AddScoped<IResourceRepository<Book, string>, MongoEntityRepository<Book, string>>();
```
does not work with JADNC v4-final. It needs to be registered on `IResourceRepository<>`, `IResourceReadRepository<>` and `IResourceWriteRepository<>`. We have a convenient helper extension method for that:
```c#
services.AddResourceRepository<MongoEntityRepository<Book, string>>();
```